script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_2.txt

```diff
@@ -45,7 +45,7 @@ Her voice slips into my ear, not from the front, but from right behind me, as if
 <Page15>
 Her breathing comes closer. Our legs tangle together. She wraps her arm around my back to block my escape.
 ―――In my field of vision.
-The strangely captivating, smooth neck of a woman―――
+This woman's strangely captivating, smooth neck―――  //"Smooth neck of a woman" sounds REALLY weird.
```

Comment:
I'd revert this. I spent a while trying to explain why from a technical standpoint, but at the end of the day the original reads more poetic to me. 
> 妙に艶めかしく、滑るような、女の首が―――

Reply:
Fair point, I assume (spoilers) that she's trying to confirm whether he's a vampire here, which is why the narration is focusing so heavily on her neck. Looking at it again, I kinda like the poetic quality of the original TL.